src/VectorizeLoops.cpp

```diff
@@ -364,22 +377,26 @@ class PredicateLoadStore : public IRMutator {
     }
 };
 
+struct VectorizedVar {
+    string name;
+    Expr min;
+    int lanes;
+};
+
 // Substitutes a vector for a scalar var in a Stmt. Used on the
 // body of every vectorized loop.
 class VectorSubs : public IRMutator {
-    // The var we're vectorizing
-    string var;
+    std::vector<VectorizedVar> vectorized_vars;
```

Comment:
When reading the code below, it seems the order of this vector is significant. Can you add a comment explaining this ordering?

Reply:
I've added comments here and in some other places in this file.